srcCpp/perftest_publisher.cxx

```diff
@@ -1683,19 +1672,7 @@ class LatencyListener : public IMessagingCB
                 if (showCpu) {
                     outputCpu = cpu.get_cpu_instant();
                 }
-                printf("One way Latency: %6lu " PERFT_TIME_UNIT
-                       " Ave %6.0lf " PERFT_TIME_UNIT
-                       " Std %6.1lf " PERFT_TIME_UNIT
-                       " Min %6lu " PERFT_TIME_UNIT
-                       " Max %6lu " PERFT_TIME_UNIT
-                       " %s\n",
-                        latency,
-                        latency_ave,
-                        latency_std,
-                        latency_min,
-                        latency_max,
-                        outputCpu.c_str()
-                );
+                _printer->print_latency_interval(latency, latency_ave, latency_std, latency_min, latency_max, outputCpu);
```

Comment:
One parameter per line, this goes beyond 80 columns.

Reply:
ok

Reply:
ok